### ISSUE TITLE
File globs are currently getting overridden

### ISSUE THREAD

**Issue by pspeter3** [TypeStrong/atom-typescript#135]
Some packages have `.d.ts` files in node modules and the default glob doesn't work. I've edited it a few times to include my files but it continues to override it.

```json
{
    "filesGlob": [
        "./**/*.ts",
        "!./node_modules/**/*.ts"
    ]
}
```

**Comment by pspeter3** [comment]
The trick is to make sure you provide the files key

**Comment by basarat** [comment]
Cool. We should only override if there is no files and no filesGlob

**Comment by basarat** [comment]
@pspeter3 thanks for testing mate. Published fix with `0.74.0` :heart_decoration:
